### ISSUE TITLE
Weird Blocks Borders

### ISSUE THREAD

**Issue by kxxzo** [Redi2Go/PhotonicEngine#71]
# Minecraft with this list of mods :

- **Photonics** <sub>_(Latest Version)_</sub>
- **Mod Menu** <sub>_(Latest Version)_</sub>
- **Fabric API** <sub>_(Latest Version)_</sub>
- **BSL** <sub>_(v8.2.09)_</sub>

## Exemple of Borders
![Proof](https://github.com/user-attachments/assets/cf7edb13-7efa-4bb4-9faa-31789f7e1bc2)

**Comment by Redi2Go** [comment]
Yeah...this is kind of a limitation of the way I render blocks. Could you please create a list that contains all the blocks that render with such weird borders? This would be an immense help in the process of manually fixing those blocks. Thanks!

**Comment by kxxzo** [comment]
Only **_<ins>full blocks</ins>_**, **_<ins>stairs</ins>_**, **_<ins>fences</ins>_** and **_<ins>slabs_</ins>** have border but there is **_<ins>`exeptions`</ins>_**

### Map with all blocks:
**_[Every Block (Custom).zip](https://github.com/user-attachments/files/17638620/Every.Block.Custom.zip)_**

Put it in `C:\Users\%USERNAME%\AppData\Roaming\.minecraft\saves` _(can copy paste)_

**Comment by Redi2Go** [comment]
The only real way of fixing this issue is authoring custom block models for each block, which is a lot of work.

The way I will fix this, is to have all the blocks you mention rendered using vanilla rendering. Specific blocks that look really cool when 3d-rendered (such as ladders, rails, brewing stands, ...) will use the 3d-rendering.
You can manually enable/disable 3d rendering for blocks you like.

I'm currently working on that, the fix will be in the next Photonics release.

Thank you so much for the effort you put into collecting the list
